itools-packager/src/main/java/com/powsybl/itools/ItoolsPackagerMojo.java

```diff
@@ -194,9 +288,16 @@ public void execute() {
             Files.createDirectories(libDir);
             copyFiles(copyToLib, libDir);
 
-            getLog().info("Zip package");
+            // create package
             String archiveNameNotNull = archiveName != null ? archiveName : packageNameNotNull;
-            zip(packageDir, targetDir, targetDir.resolve(archiveNameNotNull + ".zip"));
+            int lastIndexOf = archiveNameNotNull.lastIndexOf(".");
+            if (lastIndexOf < 0) {
+                // file extension: zip by default
+                archiveNameNotNull += ".zip";
+            }
+
+            createPackage(packageDir, targetDir, archiveNameNotNull, packageNameNotNull);
```

Comment:
You don't need to pass both archiveName and packageName. Only the archiveName is needed.

Reply:
In my tests, I assumed that if archiveName does not contain any string, archiveName will be retrieved from packageName.
With the archiveName string never empty, I removed packageName.